frontend/src/containers/jobs.js

```diff
@@ -66,7 +78,7 @@ class Jobs extends Component {
                                 <tbody>
                                     {this.props.jobs.map((job) => {
                                         return (
-                                            <tr key={job.ID}>
+                                            <tr key={job.ID} className={this.jobStatusToColor(job.Status)}>
```

Comment:
I'm torn.. going to /jobs and not seeing anything non-green is a super fast way to get an overview.. though the pattern could be green (which we should expect is the biggest number of jobs) are no color, and other job states get colorized :)